lib/Doctrine/DBAL/Portability/Connection.php

```diff
@@ -71,11 +71,11 @@ public function connect()
                 } elseif ($this->getDatabasePlatform()->getName() === "sqlite") {
                     $params['portability'] = $params['portability'] & self::PORTABILITY_SQLITE;
                 } elseif ($this->getDatabasePlatform()->getName() === "drizzle") {
```

Comment:
Note for the next person that has to maintain this: we're sorry about this massive chain of doom and `elseif` :-(